### PR TITLE
Implement Pose3d with common widget pose

### PR DESCRIPTION
**Pull request by AzulRadio**
* Part of https://github.com/gazebosim/gz-gui/issues/310

# New feature

## Summary

Implement `Pose` in Component Inspector with common widget GzPose introduced in gz-gui#424. 

Behavior should be exactly the same as before.

image

## Checklist
- [x] Signed all commits for DCO
- [x] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: src/gui/plugins/component_inspector/Pose3d.qml [hunk 1]
```diff
@@ -160,166 +86,52 @@ Rectangle {
...
-          leftPadding: 5
-          color: Material.theme == Material.Light ? "#444444" : "#bbbbbb"
-          font.pointSize: 12
+          Layout.preferredWidth: margin + indentation
         }
 
-        Item {
+        // Content
+        GzPose {
+          id: gzPoseContent
```
Comment (jennuine) [inline src/gui/plugins/component_inspector/Pose3d.qml:105]: This `id` is already used in `GzPose`, let's rename this to something else

File: src/gui/plugins/component_inspector/Pose3d.qml [hunk 2]
```diff
@@ -160,166 +86,52 @@ Rectangle {
         }
       }
     }
-
-    // Content
     Rectangle {
       id: content
```
Comment (jennuine) [inline src/gui/plugins/component_inspector/Pose3d.qml:90]: This isn't used anymore
```suggestion
```

**Review by jennuine — APPROVED** [review]

**Comment by scpeters** [comment]
are there any plans to port this change to fortress? there are conflicts when I try to merge this forward, and I'm not sure how to resolve them

**Comment by scpeters** [comment]
> @scpeters Fortress has a plotting feature for pose that is not supported in Citadel so there are conflicts when you forward-port. I am working a PR in gz-gui6 to add the feature and after a release for gui6 we should be good to forward-port.

thank you!

**Pull request merged**